README.md

```diff
@@ -15,3 +17,12 @@ Each template can be customized and updated to represent the most accurate taxon
 2. Answer the brief on-screen prompts. 
    
 More details to explain valid answers to the prompts soon!
+
+### "Parameters" way
```

Comment:
Add description and example for the valid source path structure: You must use `./` to prefix your path from your current location. And don't end your path the a forward slash `/'.

Correct source path:
```
[~/Projects/_AdobeIO_Sites]$ npx https://github.com/AdobeDocs/frontmatters.git --ignore-existing -r uxp-xd -s ./uxp-xd/src/pages
```

Incorrect source path:
```
[~/Projects/_AdobeIO_Sites]$ npx https://github.com/AdobeDocs/frontmatters.git --ignore-existing -r uxp-xd -s /uxp-xd/src/pages/
```